### PR TITLE
fix(CreateChatView): Move `ActivityCenterPopup` to appmain

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #6345

### What does the PR do

Move ActivityCenterPopup to AppMain

### Affected areas

ActivityCenter

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### CODE REVIEW

File: ui/app/AppLayouts/Chat/views/CreateChatView.qml [hunk 1]
```diff
@@ -12,13 +12,14 @@ import StatusQ.Core.Theme 0.1
 import utils 1.0
 import shared.status 1.0
 
+import "../popups"
```
Comment (alexandraB99) [inline ui/app/AppLayouts/Chat/views/CreateChatView.qml:15]: is this import really needed?

File: ui/app/mainui/AppMain.qml [hunk 2]
```diff
@@ -761,6 +766,20 @@ Item {
             }
         }
 
+        Component {
+            id: activityCenterPopupComponent
+            ActivityCenterPopup {
+                id: activityCenter
+                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
+                y: 56
```
Comment (alexandraB99) [inline ui/app/mainui/AppMain.qml:774]: this will be problematic when resizing the window, can't we use margins?

Comment (borismelnik) [reply ui/app/mainui/AppMain.qml:774]: @alexandraB99 I actually don't know, I can try :)

**Review by jrainville — COMMENTED** [review]
Code looks good. I'll just let you address Alexandra's comments before approving

File: ui/app/mainui/AppMain.qml (hunk 2 above)
Comment (borismelnik) [reply ui/app/mainui/AppMain.qml:774]: @alexandraB99 unfortunately, we can't cos `Popup` doesn't have `anchors.topMargin` or `anchors.bottomMargin`.
I tested it it works fine.

**Review by jrainville — APPROVED** [review]

**Review by alexandraB99 — APPROVED** [review]

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**